### PR TITLE
Enable indent rule and fix indent

### PR DESCRIPTION
**Pull request by SimonBrandner**
This enables the ESLint indent rule and fixes the indentation. In some places, I've fixed the indentation myself, in others, it was done by ESlint. Honestly, there wasn't much logic to where I did it myself and where not.

### CODE REVIEW

File: src/components/views/dialogs/security/RestoreKeyBackupDialog.js [hunk 1]
```diff
@@ -436,14 +436,14 @@ export default class RestoreKeyBackupDialog extends React.PureComponent {
                 {_t(
                     "If you've forgotten your Security Key you can "+
                     "<button>set up new recovery options</button>"
-                , {}, {
+                    , {}, {
                     button: s => <AccessibleButton className="mx_linkButton"
```
Comment (jryans) [inline src/components/views/dialogs/security/RestoreKeyBackupDialog.js:440]: Hmm, I guess this object's fields should now be indented further?

File: src/components/views/elements/EditableItemList.js [hunk 2]
```diff
@@ -122,10 +128,10 @@ export default class EditableItemList extends React.Component {
     _renderNewItemField() {
         return (
             <form onSubmit={this._onItemAdded} autoComplete="off"
-                  noValidate={true} className="mx_EditableItemList_newItem">
+                noValidate={true} className="mx_EditableItemList_newItem">
```
Comment (jryans) [inline src/components/views/elements/EditableItemList.js:131]: Maybe nudge `>` to a blank line and back one level?

File: src/components/views/messages/MKeyVerificationConclusion.js [hunk 3]
```diff
@@ -83,8 +83,8 @@ export default class MKeyVerificationConclusion extends React.Component {
 
         // User isn't actually verified
         if (!MatrixClientPeg.get()
-                            .checkUserTrust(request.otherUserId)
-                            .isCrossSigningVerified()) {
+            .checkUserTrust(request.otherUserId)
+            .isCrossSigningVerified()) {
```
Comment (jryans) [inline src/components/views/messages/MKeyVerificationConclusion.js:87]: Looks strange, maybe extract to a constant...?

File: src/components/views/rooms/ReadReceiptMarker.js [hunk 4]
```diff
@@ -188,7 +188,7 @@ export default class ReadReceiptMarker extends React.PureComponent {
 
         return (
             <NodeAnimator
-                    startStyles={this.state.startStyles} >
+                startStyles={this.state.startStyles} >
```
Comment (jryans) [inline src/components/views/rooms/ReadReceiptMarker.js:191]: Maybe nudge `>` to a blank line and back one level?

File: src/indexing/EventIndex.js [hunk 5]
```diff
@@ -777,7 +786,7 @@ export default class EventIndex extends EventEmitter {
      * timeline, false otherwise.
      */
     async populateFileTimeline(timelineSet, timeline, room, limit = 10,
-                               fromEvent = null, direction = EventTimeline.BACKWARDS) {
+        fromEvent = null, direction = EventTimeline.BACKWARDS) {
```
Comment (jryans) [inline src/indexing/EventIndex.js:789]: Maybe move `) {` to it own line and back a level?

File: src/stores/CustomRoomTagStore.js [hunk 6]
```diff
@@ -125,14 +125,14 @@ class CustomRoomTagStore extends EventEmitter {
                     this._setState({tags});
                 }
             }
-            break;
+                break;
```
Comment (jryans) [inline src/stores/CustomRoomTagStore.js:128]: Hmm, perhaps move inside the case blocks...?

File: src/utils/MegolmExportEncryption.js [hunk 7]
```diff
@@ -311,7 +311,7 @@ function unpackMegolmKeyFile(data) {
     while (1) {
         const lineEnd = fileStr.indexOf('\n', lineStart);
         const line = fileStr.slice(lineStart, lineEnd < 0 ? undefined : lineEnd)
-              .trim();
+            .trim();
```
Comment (jryans) [inline src/utils/MegolmExportEncryption.js:314]: Could fit on the line above...

File: test/components/views/elements/MemberEventListSummary-test.js [hunk 8]
```diff
@@ -246,8 +246,8 @@ describe('MemberEventListSummary', function() {
     });
 
     it('truncates multiple sequences of repetitions with other events between',
-    function() {
-        const events = generateEvents([
+        function() {
```
Comment (jryans) [inline test/components/views/elements/MemberEventListSummary-test.js:249]: Maybe move `function` to the `it` line above, and then de-dent again...?

File: test/components/views/elements/MemberEventListSummary-test.js [hunk 9]
```diff
@@ -396,8 +397,8 @@ describe('MemberEventListSummary', function() {
     });
 
     it('correctly orders sequences of transitions by the order of their first event',
-    function() {
-        const events = generateEvents([
+        function() {
```
Comment (jryans) [inline test/components/views/elements/MemberEventListSummary-test.js:400]: Maybe move `function` to the `it` line above, and then de-dent again...?

File: test/components/views/elements/MemberEventListSummary-test.js [hunk 10]
```diff
@@ -569,8 +571,8 @@ describe('MemberEventListSummary', function() {
     });
 
     it('handles invitation plurals correctly when there are multiple invites',
-    function() {
-        const events = generateEvents([
+        function() {
```
Comment (jryans) [inline test/components/views/elements/MemberEventListSummary-test.js:574]: Maybe move `function` to the `it` line above, and then de-dent again...?

**Review by jryans — CHANGES_REQUESTED** [review]
A good first pass, thanks for working on this! 😄

**Comment by SimonBrandner** [comment]
Thank you for the review! Everything should be hopefully fixed now

File: src/components/structures/GroupView.js [hunk 11]
```diff
@@ -43,15 +43,15 @@ import {mediaFromMxc} from "../../customisations/Media";
...
-`<h1>HTML for your community's page</h1>
-<p>
-    Use the long description to introduce new members to the community, or distribute
-    some important <a href="foo">links</a>
-</p>
-<p>
-    You can even add images with Matrix URLs <img src="mxc://url" />
-</p>
-`);
+    "<h1>HTML for your community's page</h1>" +
```
Comment (jryans) [inline src/components/structures/GroupView.js:46]: Hmm, this change seems to have altered the English string, which means it will need to be retranslated... I'd say let's revert this change, since I would expect this UI to go away soonish when Spaces takes over, and so there's not much value in having a string for Communities go through translation again.

**Review by jryans — CHANGES_REQUESTED** [review]
Almost there, one small issue.

**Review by jryans — APPROVED** [review]
Looks great, thanks again for working on this! 😄
